change/office-ui-fabric-react-2020-03-10-20-59-00-keco-remove-withresponsivemode-default-render.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "patch",
+  "comment": "Remove default ResponsiveMode causing unnecessary render",
```

Comment:
Perhaps saying which utility this is in would clarify this comment in the release notes